backend/dal/Services/Admin/Concrete/WorkflowService.cs

```diff
@@ -51,6 +51,23 @@ public Workflow Get(int id)
             return this.Context.Workflows.Find(id) ?? throw new KeyNotFoundException();
         }
 
+        /// <summary>
+        /// Get an array of workflows that are associated with the specified project 'code'.
+        /// </summary>
+        /// <param name="code"></param>
+        /// <returns></returns>
+        public IEnumerable<Workflow> GetForStatus(string code)
+        {
+            return (
+                from s in this.Context.ProjectStatus
```

Comment:
reason for the different sql query format?

Reply:
fluent annotation just makes certain queries easier to write and read.